lib/generate-plugin/index.js

```diff
@@ -1,5 +1,6 @@
 const yeoman = require("yeoman-environment");
-const PluginGenerator = require("../generators/plugin-generator").PluginGenerator;
+const PluginGenerator = require("../generators/plugin-generator")
```

Comment:
This can be destructured :)
```js
const { PluginGenerator } = require("../generators/plugin-generator")
```

Reply:
Im not done

Reply:
yeah and in the meantime we give feedback while you develop :)

Reply:
Gotcha, just don't want you to do extra work 🙌🤙